test/Serilog.Extensions.Logging.File.Tests/Serilog.Extensions.Logging.File.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
```

Comment:
Could even go for `net4.8` and `net5.0` or `net6.0`?

Reply:
Same here 🙂

https://github.com/serilog/serilog-extensions-logging/blob/dev/test/Serilog.Extensions.Logging.Tests/Serilog.Extensions.Logging.Tests.csproj#L4

Reply:
As far as I can see, the versioning/dependency management policy in `Serilog.***` is (unfortunately) messy.

Reply:
Letting each package evolve independently has some benefits, too - a lot of packages change only very rarely, so we keep work to a minimum by leaving well enough alone :-)

Once we've narrowed the Serilog organisation back down to just a few core packages, we might try pulling them back into a monorepo and doing this differently (see https://github.com/serilog/serilog/issues/1627)

Reply:
Thanks for #1627! Good news) 👍

---
I've changed tests project TFM to just `netstandard2.0`. Project uses Test SDK 15.x, AppVeyor config uses VS 15.x (2017). And there are no dependencies specific for new TFMs.